docs/node-operators/hardforks.mdx

```diff
@@ -9,39 +16,44 @@ A hard fork occurs when there are major changes to the core protocol that requir
 
 ## Overview
 
-A fork in a blockchain happens when the rules that determine which blocks are valid or invalid change. There are two types of forks: hard forks and soft forks.
+A fork in a blockchain happens when the rules that determine which blocks are valid or invalid change. 
 
-A soft fork occurs when the new rules are still compatible with the old rules. This means users can still run an old version of the node software and still connect to the network.
+There are two types of forks: soft forks and hard forks.
 
-A hard fork means the old chain is incompatible with the new chain. An example of a hard fork is the Bitcoin Cash network, which used the original Bitcoin blockchain and then hard forked to a new network that is incompatible. As seen in this example, after a hard fork it is possible for two chains to exist. To avoid splitting the chain it is important that every node on the network updates to the latest software.
+- A soft fork occurs when the new rules are still compatible with the old rules. This means users can run an old version of the node software and still connect to the network.
+- A hard fork means the old chain is incompatible with the new chain. After a hard fork it is possible for two chains to exist. To avoid splitting the chain it is important that every node on the network updates to the latest software. For example, Bitcoin Cash is considered to be the most successful hard fork of the primary cryptocurrency.
 
 ## Process
 
 Participating in a hard fork is incredibly simple. After a hard fork is announced, the first step is to update your node to the latest software. Next, you simply need to run your node and wait until the configured network start time. After the scheduled network start time, your node will sync to the latest chain and eventually a new block will be produced and the network will continue like usual.
 
 ### Updating your node
 
-When the hard fork version is announced, first shut off any node(s) you have running.
+When the hard fork version is announced:
 
-Next, remove your `.mina-config` folder since the fork will require syncing from scratch.
+1. Shut off any node or nodes you have running.
+1. Remove your `.mina-config` folder since the fork requires syncing from scratch:
 
-```
-rm -rf ~/.mina-config
-```
+    ```
+    rm -rf ~/.mina-config
+    ```
 
-Now you can follow the instructions on the [Connecting to the Network](connecting-to-the-network) to install the latest version of the daemon.
+1. Install the latest version of the daemon. 
```

Comment:
All of these steps are `1.`

Reply:
true! Best practice for docs is to use "lazy numbering" because the enumeration auto-resolves on the preview https://docs2-git-style-1fix-minadocs.vercel.app/node-operators/hardforks#updating-your-node 

pretty cool to have your attention on docs @nholland94